nautobot/docs/additional-features/jobs.md

```diff
@@ -701,27 +692,41 @@ if "job_logs" in settings.DATABASES:
 Replicate the behavior of `run_job_for_testing` manually so that your test execution most closely resembles the way the celery worker would run the test:
 
 ```python
-import uuid
 from django.contrib.auth import get_user_model
 from django.contrib.contenttypes.models import ContentType
 
-from nautobot.extras.context_managers import web_request_context
-from nautobot.extras.jobs import run_job
-from nautobot.extras.models import JobResult, Job
-
-def run_job_for_testing(job, data=None, commit=True, username="test-user"):
-    if data is None:
-        data = {}
-    user_model = get_user_model()
-    user, _ = user_model.objects.get_or_create(username=username, is_superuser=True, password="password")
-    job_result = JobResult.objects.create(
-        name=job.class_path,
-        obj_type=ContentType.objects.get_for_model(Job),
-        user=user,
-        task_id=uuid.uuid4(),
+from nautobot.extras.models import JobResult
+
+User = get_user_model()
+
+def run_job_for_testing(job, username="test-user", profile=False, **kwargs):
```

Comment:
Probably better to remove this entire section of the docs (which is about how to fake the `run_job_for_testing` function in old versions of Nautobot, and so shouldn't be updated for 2.0) at this point.